skaha/src/main/resources/add-user/skaha-add-user-keel.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{skaha.adduser.jobname}"
+spec:
+  ttlSecondsAfterFinished: 3600
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: "skaha-add-user"
+        image: images.canfar.net/skaha-system/add-user:1.2
+        imagePullPolicy: Always
+        # Userid for allocation goes in this argument.
+        # Second argument is user quota in GB
+        # TODO: automate the setting of this in the calling script
+        command: ["/usr/bin/add-user"]
+        args: ["{skaha.userid}", "{skaha.userquotagb}"]
+        volumeMounts:
+        - mountPath: "/config"
+          name: add-user-config
+        - mountPath: /root/.ssl/
+          name: servops-cert
+          readOnly: true
+        - mountPath: "/arc"
+          name: cavern-volume
+          subPath: cavern
+        - mountPath: /var/lib/sss/pipes
+          name: sssd-dir
+          readOnly: true
+      securityContext:
+        runAsUser: 0
+      serviceAccountName: skaha
+      volumes:
+      - name: add-user-config
+        configMap:
+          name: add-user-config
+      - name: servops-cert
+        secret:
+          defaultMode: 420
+          secretName: servops-clientcert
+      - name: cavern-volume
+        cephfs:
+          monitors:
+          - 10.30.201.3:6789
+          - 10.30.202.3:6789
+          - 10.30.203.3:6789
+          path: "{skaha.cephfs.path}"
+          user: "{skaha.cephfs.user}"
```

Comment:
Okay, I see now why you wanted those values.  Couldn't this yaml file be put in as a configmap (like config/launch-carta.yaml for example) and run like those ones are?  (see PostAction.createSession())

Reply:
That's a good idea.  I'll make use of that.

Reply:
This is now a script in the `/config` folder to be accessed by Skaha.